.travis.yml

```diff
@@ -3,7 +3,6 @@
 language: python
 
 python:
-  - 2.7
```

Comment:
Wondering about timing of this change given this library is used everywhere, including Insights, which is not yet on Python 3.  May be fine to constrain in Insights, as long as we don't run into any security issues, etc.
@jmbowman: Any thoughts on whether it is better to wait or it is fine?

Reply:
The last time this was discussed, we decided it was ok to drop Python 2.7 support for libraries used in Insights, since it can just avoid upgrading them.  If for some reason we do need Insights to get a fixed version of the package before it finishes the Python 3 upgrade, we can branch from the commit before this merges and do a new 2.x release.

Reply:
👍